news/templates/news/admin_events_search.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'web/base.html' %}
+{% load static %}
+{% load i18n %}
+
+{% block extra_head %}
+    <link rel="stylesheet" href="{% static 'news/css/admin_events_search.css' %}"/>
+{% endblock extra_head %}
+
+{% block body %}
+    <h1 class="ui centered header">{% translate "Check if a user has attended or is registered for an event" %}</h1>
+    
+    <div class="ui text container">
+    <form method="POST" class="ui small form">{% csrf_token %}
+        <div class="eight wide field">
+             <label>{{ form.name.label }}
+                <span data-content="{% translate "Valid arguments are username, firstname or lastname" %}">
+                    <i class="ui make-col-yellow question circle icon"></i>
+                </span>
+            </label>
+            {{ form.name }}
+        </div>
+        <button class="ui make-col-blue button" type='submit'>
+            {% translate "Search" %}
+        </button>
+    </form>
+    <br>
+    {% if users %}
+        {% for user in users %}
+            <ul id="found-user">
+                <h1>{{ user.get_full_name}}</h1>
+                    {% for ticket in user.tickets%}
+                        {% if ticket.active %}
+                            {% if ticket.timeplace %}
+                                <a href="{% url 'timeplace_ticket_list' ticket.timeplace.pk %}" target="_blank">{{ ticket.timeplace.event.title }}:
```

Comment:
CodeQL is issuing a warning about using `target="_blank"` without  noopener/noreferrer. Adding `rel="noopener noreferrer"` solves it
[https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable](https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable)